src/components/numbered-code-snippet/numbered-code-snippet.js

```diff
@@ -106,7 +114,7 @@ export default class NumberedCodeSnippet extends React.PureComponent {
       overlayElement.style.opacity = '1';
       copyElement.style.opacity = '1';
     }
-  }, 300);
+  }, 10);
```

Comment:
@katydecorah when this was set to `300`, there was a very noticeable delay of the blue bar on the left side moving to the relevant lines when expanding/collapsing the grayed out lines. Do you think we need to use `debounce` when expanding/collapsing lines (especially since we won't need to auto scroll to the first active chunk anymore)?

Reply:
Oh I have an idea. 💡

Reply:
@katydecorah @danswick I think `debounce` was introduced in the mr-ui code snippet component specifically for recalculating the position of elements when the window is resized. I'm not sure we need to use it when toggling between languages or expanding/collapsing lines since it each of those triggers happen only once. What do you think of an approach like this: https://github.com/mapbox/dr-ui/pull/206/commits/30db2d675a1370ae6ce7a2647c0aae8a5fbe4cef?

Reply:
@colleenmcginnis your solution is super smooth! Looks good to me!

Reply:
🏄‍♀